docs/conf.py

```diff
@@ -169,10 +226,9 @@
 html_static_path = ['_static']
 
 html_context = {
-        'css_files': [
-            '_static/css/fiware_readthedocs.css', #logo
-            ],
+        'css_files': select_css(html_build_dir),
         }
+print(html_context)
```

Comment:
I'd remove this log and add one in select_css showing which style files are being used